enterprise_access/apps/content_assignments/api.py

```diff
@@ -525,6 +525,27 @@ def _get_content_title(assignment_configuration, content_key):
     return content_metadata.get('content_title')
 
 
+def _get_parent_content_key(assignment_configuration, content_key):
+    """
+    Helper to retrieve (from cache) the parent content key of a content_key's content_metadata.
+    Note: content_key is either a course run key or a course key. Only course run keys have a
+    parent course key.
+
+    If content_key is for a course key, this will return the same key. Otherwise, the content_key
+    represents a course run, and this will return the run's parent course key.
+    """
+    content_metadata = _get_content_summary(assignment_configuration, content_key)
```

Comment:
nit: maybe name this `course_content_metadata` to make it clear that `_get_content_summary()` returns a course, and not a run, even if we pass a course run key into it.